sonar-project.properties

```diff
@@ -10,4 +10,5 @@ sonar.sources=man/,gen/
 sonar.cfamily.build-wrapper-output=build_wrapper_output_directory
 sonar.sourceEncoding=UTF-8
 sonar.cfamily.cache.enabled=false
-sonar.cfamily.threads=2
\ No newline at end of file
+sonar.cfamily.threads=2
+sonar.cfamily.gcov.reportsPath=gcov/
```

Comment:
I'm surprised this line being here was taken by sonarcloud since in the yaml fileI thought it was only supposed to use the settings in the yaml. I am going to copy this path into the yml settings too if you don't have a problem with it. @ahouse13

Reply:
the YAML probably sounds like the more appropriate spot for it if that's where the other settings are being generated